includes/functions.php

```diff
@@ -3451,4 +3451,106 @@ function erp_disable_mysql_strict_mode() {
 
     $wpdb->query( "SET SESSION SQL_MODE=''" );
     $wpdb->query( "SET SQL_BIG_SELECTS=1" );
-}
\ No newline at end of file
+}
+
+/**
+ * Reset ERP Data
+ *
+ * Remove Whole ERP Tables, Roles, Options
+ * Deactivate and Activate Wp ERP & ERP-Pro
+ *
+ * @since 1.8.8
+ *
+ * @return boolean|object true|WP_Error
+ */
+function erp_reset_data() {
+    global $wpdb;
+
+    try {
+        $wpdb->query('START TRANSACTION');
+
+        $options = [
+            'wp_erp_version', 'wp_erp_db_version', 'erp_modules',
+            'erp_email_settings_employee-welcome', 'erp_email_settings_new-leave-request',
+            'erp_email_settings_approved-leave-request', 'erp_email_settings_rejected-leave-request',
+            'erp_email_settings_new-task-assigned', 'erp_setup_wizard_ran', 'erp_settings_general',
+            'erp_settings_accounting', 'erp_settings_erp-hr_workdays', 'wp_erp_activation_dismiss',
+            '_erp_admin_menu', '_erp_adminbar_menu', 'erp_settings_erp-email_general', 'erp_settings_erp-email_smtp',
+            '_erp_company', 'erp_settings_erp-crm_subscription', 'erp_acct_new_ledgers',
+            'erp_email_settings_new-contact-assigned', 'erp_email_settings_hiring-anniversary-wish',
+            'wp_erp_install_date', 'widget_erp-subscription-from-widget', 'erp_tracking_notice'
+        ];
+
+        $roles = [
+            'erp_hr_manager', 'employee',
+            'erp_crm_manager', 'erp_crm_agent',
+            'erp_ac_manager', 'erp_ac_agency'
+        ];
+
+        $tables = $wpdb->get_results(
+            "SELECT TABLE_NAME FROM information_schema.TABLES
+            WHERE TABLE_SCHEMA = '{$wpdb->dbname}'
+            AND TABLE_NAME LIKE '%_erp_%'
+            AND TABLE_NAME NOT LIKE '%_erp_audit_log%'"
```

Comment:
It would be more convenient if we don't use the `%` operator at the beginning. Also, why the `audit_log` table is encoded with `%` operator? We know the exact table name. For example, this query can be rewritten in the following form:
```
            SELECT TABLE_NAME FROM information_schema.TABLES
            WHERE TABLE_SCHEMA = '{$wpdb->dbname}'
            AND TABLE_NAME LIKE '{$wpdb->prefix}_erp_%'
            AND TABLE_NAME NOT LIKE '{$wpdb->prefix}_erp_audit_log'
```